concepts/generators/introduction.md

```diff
@@ -1,2 +1,96 @@
-#TODO: Add introduction for this concept.
+# Instructions
 
+A generator in Python is a _callable function_ that returns a [lazy iterator](https://en.wikipedia.org/wiki/Lazy_evaluation).

```

Comment:
If this is meant to be the concept introduction, I think it should be shorter.  Something along the lines of this [`dicts`](https://github.com/exercism/python/blob/main/concepts/dicts/introduction.md) one.  I think usage examples are better in `about.md` and or the exercise introduction itself.

Reply:
I really like this content tho.  I just think that the introduction for a concept should be as condensed as we can make it.